source/linear-algebra/source/03-AT/06.ptx

```diff
@@ -162,31 +162,94 @@ Which of these completes the following definition for a bijective linear map
 </fact>
 
 
-
 <activity>
     <introduction>
         <p>
-The matrix space <m>M_{2,2}=\left\{\left[\begin{array}{cc}
+Consider the matrix space <m>M_{2,2}=\left\{\left[\begin{array}{cc}
     a&amp;b\\c&amp;d
-\end{array}\right]\middle| a,b,c,d\in\IR\right\}</m> has the basis
+\end{array}\right]\middle| a,b,c,d\in\IR\right\}</m> and the following matrix equation:
 <me>
-\left\{
-\left[\begin{array}{cc}
+x_1\left[\begin{array}{cc}
     1&amp;0\\0&amp;0
-\end{array}\right],
-\left[\begin{array}{cc}
+\end{array}\right]+x_2\left[\begin{array}{cc}
     0&amp;1\\0&amp;0
-\end{array}\right],
-\left[\begin{array}{cc}
+\end{array}\right]+x_3\left[\begin{array}{cc}
     0&amp;0\\1&amp;0
-\end{array}\right],
-\left[\begin{array}{cc}
+\end{array}\right]+x_4\left[\begin{array}{cc}
     0&amp;0\\0&amp;1
-\end{array}\right]
-\right\}
-</me>.
+\end{array}\right]=\left[\begin{array}{cc}
+2&amp;-3\\0&amp;5
+\end{array}\right].
+</me>
         </p>
     </introduction>
+    <task>
+        <statement>
+            <p> How many solutions does this equation have?
+                <ol marker="A." cols="2">
+                    <li>1</li>
+                    <li>infinitely many</li>
+                    <li>none</li>
+                    <li>2</li>
+                </ol>
+            </p>
+        </statement>
+    </task>
+    <task>
+        <statement>
+            <p>
+                How many solutions do the following matrix equation have?
+                    <me>
+                        x_1\left[\begin{array}{cc}
+                            1&amp;0\\0&amp;0
+                        \end{array}\right]+x_2\left[\begin{array}{cc}
+                            0&amp;1\\0&amp;0
+                        \end{array}\right]+x_3\left[\begin{array}{cc}
+                            0&amp;0\\1&amp;0
+                        \end{array}\right]+x_4\left[\begin{array}{cc}
+                            0&amp;0\\0&amp;1
+                        \end{array}\right]=\left[\begin{array}{cc}
+                        4&amp;3\\1&amp;2
+                        \end{array}\right].
+                        </me>
+                <ol marker="A." cols="2">
+                    <li>1</li>
+                    <li>infinitely many</li>
+                    <li>none</li>
+                    <li>2</li>
+                </ol>
+            </p>
+        </statement>
+    </task>
```

Comment:
![image](https://github.com/user-attachments/assets/ea04560e-a4ae-4e71-ad1c-ff86c69a7eb6)

Minor point: the equation in the introduction probably belongs in the first task.

Major point: how do you expect students to answer these questions? They cannot convert the equation to a Euclidean vector equation (that's the point of this activity, to convince them that they can convert to $\mathbb R^4$), which puts their usual RREF approach out of reach.

But what about this? Just ask if (the basis) is a spanning set, with answer choices:

- No, [example] is not a linear combination of these matrices. (3x)
- Yes, every matrix in M22 is a linear combination of these matrices.

Next part: is (the basis) linearly independent?

- No, [matrix] is a linear combination of [matrices] (3x)
- Yes, each matrix cannot be expressed as a linear combination of other matrices in the set.

Then you're set up for part (c) without having to solve equations.

Reply:
So, what I coach them to do in the past is to simplify the LHS of that matrix equation into $$\left[\begin{array}{cc}x_1&x_2\\x_3&x_4\end{array}\right]$$ and then we're just looking at $$\left[\begin{array}{cc}x_1&x_2\\x_3&x_4\end{array}\right]=$$ some specific matrix. 

My students seem to have a hard time really processing how to build a linear combination of matrices and so I think having the LHS written out gives them a starting point.

Reply:
ultimately, I think they'll be forced do what I describe (or take a more flexible approach) with your scaffolding, so I'm tempted to give it a shot. I also think your scaffolding would lead to more diversity of discussion in a productive way.

Reply:
Oh of course. I too overlooked the obvious approach to turn [matrix]=[matrix] into a *linear system* rather than trying to find the right Euclidean vector equation. So scaffolding in that direction works as well, but I think the made up constants on the right-hand side is distracting more than building on their intuition of "spanning means we can build everything" and "independence means there's no redundant information" to approach this.

Reply:
agreed. i've got a commit coming in with the suggested changes for your review